### ISSUE TITLE
Duplicate DeepL Recipe

### ISSUE THREAD

**Issue by jutonium** [autopkg/peterkelm-recipes#3]
Hi Peter
I would kindly ask you to retract your DeepL workflow which you merged two days ago.
The download recipe is an incomplete duplicate of andredb90's downloader and we added a working munki recipe without duplicating any recipes which were reported by `autopkg search deepl` on Monday.
Best wishes
jutonium

**Comment by peterkelm** [comment]
@Jutonium, I am honestly a little surprised by your request.

Could you please explain why you believe our recipe is an "incomplete duplicate" of that andredb90's download recipe? I just compared them and cannot find much difference at all... Truth to be told, I was not even ware of its existence. We wrote our DeepL recipes on Sept 12 2019, when there wasn't anything available at all - however, we did not bother uploading them until later.

I have no general objection retiring ours and using that download recipe instead and do - of course - support the "one recipe" approach that autopkg is mandating. Yet, I am wondering why you came up with another `DeepL.munki.recipe` when ours existed already... and the suggested approach would have been a pull request, e.g. with a reference to that andredb90 download recipe.

**Comment by jutonium** [comment]
Hi Peter
We ran `autopkg search deepl` before we started to write a workflow and found only andredb90's download and filewave workflows. After merging our recipes yours started showing up as well. Maybe a caching error.
I would suggest that we deprecate and retract ours. Could you add a variable for the filename or use andredb90's downloader?
The edit would look like this:
```
<dict>
    <key>Arguments</key>
    <dict>
        <key>filename</key>
        <string>%NAME%.dmg</string>
        <key>url</key>
        <string>%DOWNLOAD_URL%</string>
    </dict>
    <key>Processor</key>
    <string>URLDownloader</string>
</dict>
```
Thank you

**Comment by peterkelm** [comment]
@jutonium, sorry that took some time. I updated our recipe to use andredb90's download recipe as requested by you.
